codalab/lib/bundle_cli.py

```diff
@@ -695,11 +734,9 @@ def do_upload_command(self, args):
             raise UsageError('Nothing to upload')
 
         # Pull out the upload bundle type from the arguments and validate it.
-        if args.bundle_type not in UPLOADED_TYPES:
-            raise UsageError('Invalid bundle type %s (options: [%s])' % (
-              args.bundle_type, '|'.join(sorted(UPLOADED_TYPES)),
-            ))
-        bundle_subclass = get_bundle_subclass(args.bundle_type)
+        # Note: only allow dataset bundles (eventually deprecate the program bundle and just have )
+        bundle_type = 'dataset'
```

Comment:
Should we potentially just set this to `'uploaded'` (and add the appropriate fields in `UploadedBundle` to make it a concrete type)?